documentation/business-analyst-activities.md

```diff
@@ -1 +1,19 @@
 # Business Analyst Activities
+
+You must have a GitHub account and access to the Teaching Vacancies repository to be able to make any changes to the code base, or to edit/create files (e.g, .yml or .md text files)
+<Insert link here to new page> This file explains the process by which new team members can get the access they require on GitHub.
+
+## Updating Polygon Mappings in the mapped_locations.yml file
+This section details the process for updating the search mappings so that a polygon search is used in place of a point search.
```

Comment:
Not quite correct: it would only do this if (1) there is no polygon under the original name (i.e. the search doesn't use a polygon already) and (2) there *is* a polygon under the new name.

The effect of changing the file is to change the name we use to look for the polygon in the database.

So it's more accurate to say that we use this file in order to point a user-inputted search term to a polygon (which we already have).

Reply:
Thanks David, have updated accordingly.